cg/src/domains/management/subjects/subjects.service.js

```diff
@@ -2,33 +2,55 @@ const { db } = require('../../../db');
 const { decryptFromStorage } = require('../../../utils/encryption');
 const winston = require('winston');
 
+const PAGE_SIZE = 10; // This could go in constants, inside utils
+
 class SubjectsService {
   constructor(database = db) {
     this.db = database;
   }
 
-  async listSubjects() {
-    const data = await this.db('subjects')
+  async listSubjects(requestedPage) {
+    if (requestedPage == undefined) {
+      requestedPage = 1;
+    }
+    const [numberOfsubjectsObject] = await this.db('subjects')
+      .join('subject_keys', 'subjects.id', '=', 'subject_keys.subject_id')
+      .whereNotNull('personal_data')
+      .whereNotNull('key')
+      .count('personal_data');
+
+    var numberOfsubjects = numberOfsubjectsObject.count;
```

Comment:
we dont use var (the linter actually goes off I think. Or it should I think.)

Reply:
there are pre-commit hooks when committing from root project directory